user_interaction/account_pages/config.py

```diff
@@ -19,4 +19,5 @@ def get_urls(self):
             path('', SiteAccountView.as_view(config=self), name='site_account'),
             path('change-password/', AccountPasswordChangeView.as_view(config=self), name='password_change'),
             path('change-layout/', LayoutPreferencesUpdateView.as_view(config=self), name='layout_change'),
+            path('account/edit', AccountChangeView.as_view(config=self), name='account_change'),
```

Comment:
This causes a url of the form `/account/site/account/edit` (note the double "account"). We're also slowly transitioning to have all URLs should end with a trailing `/`.
```suggestion
            path('edit/', AccountChangeView.as_view(config=self), name='account_change'),
```